content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/configuring-your-minio-storage-bucket-for-github-actions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Configuring your MinIO storage bucket for GitHub Actions
+intro: 'Configure your custom MinIO storage bucket for use with {% data variables.product.prodname_actions %}.'
```

Comment:
Because this article is exclusively about setting up MinIO, it would be my preference that we instead link to the [MinIO docs for doing this](https://min.io/docs/minio/container/index.html) instead of replicating it here.

It creates issues for maintainability and supportability having this article in the GitHub docs, when this article is exclusively about MinIO configuration. Similar to the other supported storage providers, we don't include instructions for setting up Amazon/GCS buckets or Azure storage containers, but instead link to their respective docs.

Reply:
I agree with that, Lucas. Let's keep the scope of this article to what it takes to configure GitHub Enterprise Server to talk to an existing MinIO installation.